lib/Base/Tensor.cpp

```diff
@@ -501,3 +501,11 @@ void Tensor::convertToType(ElemKind newTy) {
   }
   *this = std::move(tmp);
 }
+
+uint64_t Tensor::getUnpaddedSize() {
+  if (unpaddedSize_) {
+    return unpaddedSize_;
```

Comment:
If I'm right that virtually-padded tensors are only valid if unowned, `assert` that the tensor is unowned here.

Reply:
I don't think they're virtually padded they are real padded.